addons/l10n_it_edi/models/account_move.py

```diff
@@ -359,6 +359,7 @@ def _l10n_it_edi_add_base_lines_xml_values(self, base_lines_aggregated_values, i
             # Other data.
             other_data_list = it_values['altri_dati_gestionali_list'] = []
             if base_line['currency_id'] != self.company_currency_id:
+                it_values['prezzo_unitario'] = base_line['currency_id']._convert(it_values['prezzo_unitario'], self.company_currency_id, date=self.date)
```

Comment:
It's a bit weird to be doing this in the `# Other data` section when there is a dedicated `# Price unit` section :nerd_face: